pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSTrain.java

```diff
@@ -36,14 +37,20 @@ public POSTrain() {
 
     public POSTrain(int iter) {
         this.iter = iter;
+        rm = new POSConfigurator().getDefaultConfig();
+        this.init();
+    }
+
+    public POSTrain(int iter, String configFile) throws IOException {
+        this.iter = iter;
+        rm = new POSConfigurator().getConfigFromFile(configFile);
```

Comment:
This line should be "new POSConfigurator().getConfig( new ResourceManager( configFile ) )" -- this way, the user only needs to specify non-default values in the config.